crates/gosub_interface/src/config/chrome.rs

```diff
@@ -0,0 +1,6 @@
+use crate::chrome::ChromeHandle;
+use crate::config::HasRenderBackend;
+
+pub trait HasChrome: HasRenderBackend + Sized {
```

Comment:
Please describe what chrome is and what it does...

Reply:
Yeah, I still need many comments. I am also not completely happy about the `HasChrome` thingy.